### ISSUE TITLE
Future of SANE

### ISSUE THREAD

**Issue by IanVS** [artificialio/sane#179]
@Globegitter, you've talked in the past about moving SANE to the sane org.  Is that still in the works?  It seems nobody from Artificial IO has stepped up to take over maintenance, and I think you've moved on, so I'm wondering about the future of this tool.

**Comment by Globegitter** [comment]
@IanVS Yes indeed, it has been a while now. I should hear back from Artificial shortly and will move SANE over to the open source organisation. Once that is done I am happy to talk about the future of it.

**Comment by nahtnam** [comment]
:+1: Hope this happens soon @Globegitter

**Comment by johnnybridges** [comment]
Sadly we do not have the resources to maintain SANE at Artificial without Markus. I have agreed to move the project to the SANE organisation at some point over the next few weeks. 

Apologies for the delay, but as a company we have been waiting for our R&D tax relief to be approved by HMRC. SANE was listed as one of the projects within our claim - and HMRC are yet to respond (usually takes around 6 weeks). We're reluctant to move until they have responded to avoid confusion, hence the delay in moving things over. Rest assured it will be moved and we have not forgotten. Thank you for your patience.

**Comment by kedano** [comment]
Isn't it possible to create a new github-organization that forks the whole project and continue from there?

**Comment by BransonGitomeh** [comment]
yea :+1: @kedano  on that....

**Comment by Globegitter** [comment]
Hey all, sorry for being so quiet here. Job and all has kept me more busy than I wanted and anticipated. I am glad to see the project now in the open source repo. If anyone wants to I am happy to take it over. Just ping me an email and I can give the rights here and on npmjs.com
